packages/local-environment/src/nomadenv.ts

```diff
@@ -141,6 +145,23 @@ export class NomadEnv {
     return false;
   }
 
+  get forkUrl(): string | undefined {
+    if (process.env.ALCHEMY_API_KEY)
+      return `https://eth-mainnet.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}`;
+  }
+
+  get wETHAddress(): string {
+    return `${process.env.WETH_ADDRESS}`;
+  }
+
+  get tDomain(): NomadDomain | undefined {
+    return this.domains[0];
+  }
+
+  get jDomain(): NomadDomain | undefined {
+    return this.domains[1];
+  }
+
```

Comment:
Also these guys. T and J should only exist in the Test case, they should not migrate into the business logic

Reply:
Removed

Reply:
Removed